bedrock/security/templates/security/web-bug-bounty.html

```diff
@@ -27,6 +27,51 @@ <h1 itemprop="name" class="title-shadow-box">Web and Services Bug Bounty Program
       </ul>
       <p>If two or more people report the bug together, or working independently at approximately the same time, the reward will be divided between them.</p>
 
+      <h2>How To Submit Bugs</h2>
+    
+      <p></p>The sooner we can reproduce and fix the bug, the sooner we can validate it, fix it, and send your reward.</p> 
+
+      <p>We prefer to receive bug reports in English. If English is not your native language and you are not fluent,  Mozilla has employees and volunteers from all over the world who can help. 
+If necessary, please write bug reports in your native language rather than using automated translation software. Automated translation software tends to mangle technical descriptions in indecipherable way.</p>
+
+      <p>There are three main things you can provide which will help us to evaluate your submission quickly and pay a bounty sooner:</p>
+      <ol>
+      <li>What is the attack scenario?</li>
+      <li>What is the step-by-step exploit process?</li>
+      <li>What is the security impact?</li>
+      </ol>
+      
+    <h3>What is the attack scenario?</h3>
+    
+    <p>For the attack scenario, please describe how an attacker would use the bug you are submitting, any necessary conditions for it to work, and what the attacker would gain through successful attack. Please answer the following questions to the best of your ability:</p>
+        <li>What is the necessary position of the attacker? (Example: Logged in to x website, unauthenticated, etc.)</li>
+        <li>Are there any other assumptions that must be made about the victim?  (Example: only affects specific web browser versions, requires that the user can be tricked into clicking on a link, etc. </li>
+        <li>How might a malicious actor use this attack? (Example: Steal victim’s cookies, redirect to malicious website, read files on server outside web root, remote OS command injection, XSS users, etc.)</li>
+
+      <h3>What is the step-by-step exploit process?</h3>
+      <p>Please describe the multi-step process required, step by step, in the right order. If there are multiple parties involved, please describe them all clearly. For example:</p>
+      <ol>
+      <li>Visit the url https://xyz.firefox.com/somepage/form?name=value&name2=value2</li>
+      <li>Login</li>
+      <li>Click the “Frobozz” button</li>
+      <li>Place order for “Anti-Grue Cream” to shipping location “small mailbox,” continue navigation to the Shipping Address page.</li>
+      <li>Add the following payload "><img src=x onerror=alert(document.domain);// > into the “City” field, then click     Submit.</li> 
```

Comment:
This will render this image tag. Maybe put this section in a `<pre>` or swap out all of the `<` for `&lt;` and the `>` for `&gt;`.